docs/overview.md

```diff
@@ -0,0 +1,149 @@
+# Cluster Provisioning with CAPI
```

Comment:
In case you haven't seen this: https://github.com/canonical/k8s-snap/pull/539/files.
We'll have to move our CAPI docs pages to the k8s-snap.